docs/esm.md

```diff
@@ -1,6 +1,8 @@
 # ES Modules (experimental)
```

Comment:
Do we still consider esm support experimental?

Reply:
I would say yes.